view/utils/init-table-sorting-on-client.js

```diff
@@ -34,9 +34,22 @@ module.exports = function (domElement/*, opts */) {
 		type: 'numeric'
 	});
 	var opts = normalizeOptions(arguments[1]);
-	setInterval(function () {
-		var element = window.jQuery(domElement);
-		element.trigger('update');
-		element.tablesorter(opts);
-	}, 500);
+	var additionalOpts = normalizeOptions(arguments[2]);
+
+	if (additionalOpts && additionalOpts.beforeSortStartFn) {
```

Comment:
It looks like we count on our luck here. We use `setTimeout instead of `setInterval` to setup the whole tablesorter...
If I understand correctly we introduced using `setInterval` to avoid problems with `setTimeout` (as `setTimeout` could simply be not enough)...

I don't see a reason for changing the way we setup sorter for that particular case. So what was the problem here? You didn't want bind for the second time? In other words: Will this such code work:

```javascript

setInterval(function () {
var element = window.jQuery(domElement);
if (additionalOpts && additionalOpts.beforeSortStartFn) {
 element.bind("sortStart", function () {
 additionalOpts.beforeSortStartFn(element);
});
}
element.trigger('update');
element.tablesorter(opts);
}, 500);
```